core/src/api/api.rs

```diff
@@ -180,6 +184,40 @@ impl GraphAPI for GraphState {
 		result
 	}
 
+	/// Export the graph pages for a certain user encrypted using the latest published key
+	fn force_recalculate_graphs(&self, user_id: &DsnpUserId) -> Result<Vec<Update>> {
```

Comment:
This function doesn't have a direct test since it would be an integration test and we have another ticket for it.